CryptoAnalysis/src/main/java/crypto/exceptions/CryptoAnalysisException.java

```diff
@@ -0,0 +1,10 @@
+package crypto.exceptions;
+
+public class CryptoAnalysisException extends RuntimeException {
```

Comment:
The whole idea was to get rid of the `RuntimeException` ;)
Please use `Exception` as base class